### DISCUSSION TITLE
GIMP would make a great inpainting frontend.

### DISCUSSION THREAD

**Discussion by alamonelf** [comfyanonymous/ComfyUI#509] [Ideas]
Think of the delicious goodness. A GIMP plugin that makes it a facility for ComfyUI. Obviously since it aint doin much GIMP would have to subjugate itself. This is where this is going and think of text tool inpainting. 

Just dreamin and playing.

ok TY ILY bye

**Comment by jwd-dev** [comment]
GIMP also has scripts to automate the application of filters. Could be nice if you want to pump your resulting image into something you made in GIMP.
